pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -170,7 +178,7 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 			return c.updateStatus(config, existingDeployments)
 		}
 		c.recorder.Eventf(config, kapi.EventTypeWarning, "DeploymentCreationFailed", "Couldn't deploy version %d: %s", config.Status.LatestVersion, err)
-		return fmt.Errorf("couldn't create deployment for deployment config %s: %v", deployutil.LabelForDeploymentConfig(config), err)
+		return fatalError(fmt.Sprintf("couldn't create deployment for deployment config %s: %v", deployutil.LabelForDeploymentConfig(config), err))
```

Comment:
Hm, what are the cases where the controller will be denied to create a new RC? Exceeded quota. What else? I am not sure this should be treated as fatal without more fine-grained error handling.